### PR TITLE
feat(ui): Indicate invites sent for approval in invite modal

### PR DESCRIPTION
**Pull request by slohmes**
Updates the invite modal to make it clear to members that their invite has been sent to owners / managers for approval:
* reword subheading text
* add alert box, linking to requests tab
* add extra text to status (bottom left)
* remove the little question mark next to the header
* update "send more..." button text

These changes only show up for members who don’t have invite access. Managers and owners still view the old (current) invite module display.

Part of ENT-216.

### CODE REVIEW

**Comment by slohmes** [comment]
Screen Shot 2020-06-15 at 11 38 27 AM

**Review by leedongwei — APPROVED** [review]

**Comment by EvanPurkhiser** [comment]
IMO this alert box is aggressively big. We need to do spelling about these alert sizes soon

File: src/sentry/static/sentry/app/components/modals/inviteMembersModal/index.tsx [hunk 1]
```diff
@@ -318,27 +329,26 @@ class InviteMembersModal extends AsyncComponent<Props, State> {
...
             ? t('Invite new members by email to join your organization.')
             : t(
                 `You don’t have permission to directly invite users, but we’ll
-                 send a request on your behalf.`
+                 send your request to organization managers and owners for approval.`
               )}
         </Subtext>
+        {!this.willInvite && (
+          <Alert type="info" icon="icon-circle-info">
+            {tct('Check the status of your pending invite requests in the [link].', {
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/modals/inviteMembersModal/index.tsx:343]: You can write this as ‘in the [link:Requests tab]’ and then make the link element self closing.

This gives translators better context

File: src/sentry/static/sentry/app/components/modals/inviteMembersModal/index.tsx [hunk 2]
```diff
@@ -256,7 +257,17 @@ class InviteMembersModal extends AsyncComponent<Props, State> {
       const sentCount = statuses.filter(i => i.sent).length;
       const errorCount = statuses.filter(i => i.error).length;
 
-      const invites = <strong>{tn('%s invite', '%s invites', sentCount)}</strong>;
+      const invites = (
+        <strong>
+          {this.willInvite
+            ? tn('%s invite', '%s invites', sentCount)
+            : tn(
+                '%s invite request to owners and managers for approval',
```
Comment (EvanPurkhiser) [inline src/sentry/static/sentry/app/components/modals/inviteMembersModal/index.tsx:265]: Is this wording right? Should it be “one invite request sent” ?

**Review by evanpurkhiser — CHANGES_REQUESTED** [review]

**Comment by BYK** [comment]
We would like to adjust this in single tenant installs.

**Pull request closed without merging**